htdocs/core/class/notify.class.php

```diff
@@ -24,7 +25,7 @@
  *      \brief      File of class to manage notifications
  */
 require_once DOL_DOCUMENT_ROOT.'/core/class/CMailFile.class.php';
-
+require_once DOL_DOCUMENT_ROOT.'/core/lib/functions.lib.php';
```

Comment:
Why do you need to include functions.lib.php ?
A dolibarr class should be loaded into a dolibarr context, so main or master file have been already loaded and they automatically include this file already.

Reply:
@eldy indeed. I have delete it